counsel.el

```diff
@@ -2699,6 +2719,34 @@ Note: don't use single quotes for the regex."
 (ivy-set-occur 'counsel-rg 'counsel-ag-occur)
 (ivy-set-display-transformer 'counsel-rg 'counsel-git-grep-transformer)
 
+(defcustom counsel-rg-pcre-flag 'auto
```

Comment:
Sorry to bikeshed, I'm just wondering - is it worth using the term "switch" instead of "flag" for consistency with the rest of Emacs? See [`(elisp) Tips for Defining`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Tips-for-Defining.html).

Reply:
The Ripgrep manual refers to them as flags.

Reply:
That's not really an argument - if we used each tool's terminology then nothing would be consistent - but I won't insist.